config/environments/development.rb

```diff
@@ -15,9 +15,18 @@
     config.cache_store = :null_store
   end
   config.active_storage.service = :local
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
   config.action_mailer.perform_caching = false
   config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    :domain => ENV['MAILER_DOMAIN'],
+    :password => ENV['MAILER_PASSWORD'],
+    :user_name => ENV['MAILER_USERNAME'],
+    :address => "smtp.mailgun.org",
+    :port => 587,
```

Comment:
This should be in your env file

Reply:
Now it is stored in the encrypted credentials too